project3/style.css

```diff
@@ -0,0 +1,108 @@
+body {
+    background-color: rgb(243, 240, 240);
+    margin: 0;
+    font-family: helvetica, Impact, 'Arial Narrow Bold', sans-serif;
+    font-size: 13px;
+    text-align: left;
+}
+
+h1 {
+    background-color: rgb(146, 206, 224);
+    font-size: 14px;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 20px;
+    text-align: center;
+}
+
+h2 {
+    font-size: 12px;
+    text-align: center;
+}
+
+h3 {
+    font-size: 12px;
+    text-align: center;
+}
+
+h4 {
+    font-size: 12px;
+    text-align: center;
+}
+
+.morning {
+    background-color: rgb(30, 216, 230);
+    padding: 20px;
+}
+
+.midday {
+    background-color: rgb(36, 171, 212);
+    padding: 20px;
+}
+
+.evening {
+    background-color: rgb(37, 134, 190);
+    padding: 20px;
+}
+
+img {
+    display: block;
+    height: auto;
+    max-width: 10%;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+figcaption {
+    padding-top: 1%;
+    text-align: center;
+    font-size: 12px;
+}
+
+svg {
+    display: block;
+    height: auto;
+    margin-right: auto;
+    margin-left: auto;
+    width: 4%;
+}
+
+@media (min-width: 500px) {
```

Comment:
Minimal responsive styling here. Would like to see you tackle some of the layout challenges you originally planned.